sdk/keyvault/keyvault-keys/src/localCryptography/algorithms.browser.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LocalSupportedAlgorithmsRecord } from "./models";
+
+/**
+ * This file contains the implementation of local supported algorithms for the browser.
+ *
+ * We currently don't support any cryptography operation in the browser.
+ *
+ */
+
+/**
+ * A plain object containing all of the locally supported algorithms.
+ */
+export const localSupportedAlgorithms: LocalSupportedAlgorithmsRecord = {
```

Comment:
If we keep the current types as is, this contraption is necessary. As soon as we get into browser support, this will expand and be coherent.